package.json

```diff
@@ -54,7 +54,14 @@
     "electron:build:serve": "npm run electron:build_0compile && npm run electron:build_1combineJs &&  npx electron out-electron/index.js --serve --built-serve",
     "docs:json": "compodoc -p ./tsconfig.base.json -e json -d .",
     "storybook": "npm run docs:json && start-storybook -p 6006",
-    "build-storybook": "npm run docs:json && build-storybook"
+    "build-storybook": "npm run docs:json && build-storybook",
+    "package:windows": "npm run build:prepare && npm run build:windows && node scripts/move-binary.js",
```

Comment:
since `build:prepare` and `build:<OS>` is running before the tauri builds anyway you don't need to call em again - that'll save up a good chunk of each pipelines time